FlowCrypt/AppDelegate.swift

```diff
@@ -16,6 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return true
         }
         GlobalRouter().proceed()
+        FirebaseApp.configure()
```

Comment:
Can you add some if-else meta tag around the import and also around this statement to only be done for debug builds? I'm not comfortable having this part of non-debug codebase.

Reply:
Will do